examples/k8s/service/main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/gorilla/mux"
+
+	"github.com/nevzatseferoglu/hz-go-service/util"
+)
+
+// NewHttpServer creates new server instance according to given router and service config.
+func NewHttpServer(router *mux.Router, serviceConfig *util.ServiceConfig) *http.Server {
+	return &http.Server{
+		Handler:      router,
+		Addr:         fmt.Sprintf(":%d", serviceConfig.Port),
+		ReadTimeout:  serviceConfig.Timeout,
+		WriteTimeout: serviceConfig.Timeout,
+	}
+}
+
+// NewRouter returns a new mux router includes certain handlers.
+func NewRouter(service *util.Service) *mux.Router {
+	router := mux.NewRouter()
+	router.HandleFunc("/", service.RootHandler)
+	okHandler := func(rw http.ResponseWriter, _ *http.Request) {
+		rw.WriteHeader(http.StatusOK)
+	}
+	router.HandleFunc("/health", okHandler)
+	router.HandleFunc("/readiness", okHandler)
+	router.HandleFunc(fmt.Sprintf("/%s", util.RootEndpoints[0]), service.ConfigHandler).Methods(http.MethodGet)
```

Comment:
Let's remove the `RootEndpoints` variable and write the endpoints directly for better visibility such as "/config"

Reply:
Thanks, [67e0826](https://github.com/hazelcast/hazelcast-go-client/pull/743/commits/67e08268ff2729948d21bfd76cac48242408b444).